homeassistant/core.py

```diff
@@ -114,7 +114,10 @@
 
 def split_entity_id(entity_id: str) -> list[str]:
     """Split a state entity ID into domain and object ID."""
-    return entity_id.split(".", 1)
+    try:
+        return entity_id.split(".", 1)
+    except AttributeError as err:
+        raise HomeAssistantError("Unexpected entity ID format", entity_id) from err
```

Comment:
Did you expect to include the entity ID in the message? I don't see how the second arg is used? 

Also did you verify every place where this is used to make sure no one was catching attribute error?

Reply:
FWIW, it is adding the entity_ids. Patched my running container and got this error after a long wait:

`homeassistant.exceptions.HomeAssistantError: ('Unexpected entity ID format', ['switch.grijze_lamp', 'light.window_level', 'switch.staande_lamp', 'switch.zoutlamp', 'climate.woonkamer'])`

Reply:
> Did you expect to include the entity ID in the message?

Yes, see, above. In example linked by @hmmbob 

> Also did you verify every place where this is used to make sure no one was catching attribute error?

No, but the PR can be closed as the goal of this PR was to find the issue... which has been found. It is caused by this PR: 

Send only a single event per incoming Google command
<https://github.com/home-assistant/core/pull/49449>

Reply:
TIL we can just pass tuples to exceptions 😆